sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
+maintenance of cellular integrity, metabolism, transcription/translation, and 
+cell-cell communication. Complete description of protein structures and 
+functions is a fundamental step towards understanding biological life and 
+also highly relevant in the development of therapeutics and drugs. Tons of 
+protein sequences have been generated, but fewer than 100,000 of them 
+have experimentally-solved structures. As a result, computational structure 
+prediction is essential for a majority number of protein sequences. However, 
+predicting protein 3D structures from sequence alone is very challenging, 
+especially when similar templates are not available. In the past decades, 
+various computational methods have been developed to predict protein 
+structure from different aspects, including prediction of secondary structure, 
+torsion angles, solvent accessibility, inter-residue contact map, disorder 
+regions and side-chain packing.
+
+Machine learning is extensively applied to predict protein structures and 
+some success has been achieved. For example, secondary structure can be 
+predicted with about 80% of Q3 accuracy by a 2-layer neural network 
+method PSIPRED [@doi:10.1093/bioinformatics/16.4.404]. Starting from 
+2012, deep learning has been gradually introduced to protein structure 
+prediction. The adopted deep learning models include deep belief network, 
+LSTM(long short-term memory), deep convolutional neural networks (DCNN) 
+and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1 
+@doi:10.1038/srep18962]. Here we focus on deep learning methods for 
+two representative subproblems: secondary structure prediction and 
```

Comment:
For protein contact prediction, it is true that deep learning makes the biggest progress since this problem appeared to be very challenging before 2011. Only in recent years (2014 and 2016) we saw big improvement. The first improvement comes from structure learning for co-evolutionary analysis (which mostly works on proteins with lots of sequence homologs) and the second improvement comes from my deep learning algorithm (which works on proteins with only dozens of sequence homologs). 

The reason I chose secondary structure prediction because it is a basic problem and also easy to understand. This problem is also more accessible to pure machine learning people than other subproblems such as disorder prediction and solvent accessibility prediction. Secondary structure prediction is almost an essential module of any protein structure prediction package.